.travis.yml

```diff
@@ -30,16 +30,20 @@ notifications:
       secure: "hS4q3WH3dnXCZAJ/vH9SNQIMnu/AOfdmvSkOhFGgUQE8gUyxdCSwJwPTzOYrDaykXO9GKa5mnYiCarhpQ3TUuHa5X6Ei2M2cYveNED0fwVPahObUqvIW2RmsdXllR0Wrca4JaWsZOsdOcFYKaqOPUFTHL5VZvm01voHp0WerbtB45HxzmDKBDUIDbWe8i8rKFza+hEJqFUmep0Psr2YEmocWnN69GkAQ9Bzew7iFVjicXIh8I2L2sy1dgt0lXESvH51+HT6nfGTm61kjUThEJZ8Qo217rjNLYcoxgHtPMrq4EezfM0l1U2USidu0yyFKHUY4JXH6sfxLw/Rp4Wiphgj5scHiQZ7xbDnwNmwYSmfP6vQ3JwZ5oyckcCjnSryqaD56pUkZjvmxi+OfHTJDN9FrE0kwshWcYBPCww8Uo2nlwrzNpfkxp3bV07iEYAXUIRbFj5gkpzdrp/ZhGuWzCYwRGcpqg8B1sR1+PiBIL3pm//sF0r9jWqKIPRkdlClvTZ20g1ERCzKGPDTgvjskUD0AV8rf6Jp8BIOSTc5VEIDXqIfptdrzMDKOJd2pBRkMEQVwlax1jcpMgFb22sNWqMXNaUn+39U7ORNdmQO20/76ziHr20tktBRymifASBCjrOMy7BFIlbOnkxYfwI6zUruHtTtBBsvr8NcEvl8mHQU="
 
 before_install:
-- "./tools/travis/setup.sh"
+  - "./tools/travis/setup.sh"
 install: true
 script:
-- "./tools/travis/build.sh && ./tools/travis/deploy.sh && ./tools/travis/test.sh"
+  - "./tools/travis/build.sh && ./tools/travis/deploy.sh && ./tools/travis/test.sh"
 deploy:
-- provider: script
-  script: "./tools/travis/publish.sh openwhisk ${TRAVIS_TAG%@*} ${TRAVIS_TAG##*@}"
-  on:
-    tags: true
-    all_branches: true
+  - provider: script
+    script: "./tools/travis/publish.sh openwhisk ${TRAVIS_TAG%@*} ${TRAVIS_TAG##*@}"
+    on:
+      tags: true
+      all_branches: true
+  - provider: script
+    script: "./tools/travis/publish.sh openwhisk 3.1.1 master && ./tools/travis/publish.sh openwhisk 4.1 master"
```

Comment:
Let's only do `4.1` the `3.1.1` image might not even build
```
./tools/travis/publish.sh openwhisk 4.1 master
```

Reply:
@csantanapr Should be good now.